samples/nrf_desktop/configuration/zephyr.conf

```diff
@@ -36,6 +36,9 @@ CONFIG_LOG_PRINTK=y
 CONFIG_LOG_PRINTK_MAX_STRING_LENGTH=256
 CONFIG_LOG_BUFFER_SIZE=4096
 CONFIG_LOG_BACKEND_RTT_MESSAGE_SIZE=256
+CONFIG_LOG_FUNCTION_NAME=n
+CONFIG_LOG_STRDUP_BUF_COUNT=64
+CONFIG_LOG_STRDUP_MAX_STRING=128
```

Comment:
Do we need buffers that big? It will result in over 8 kB memory usage... Please take a look at `zephyr/subsys/logging/log_core.c`

Reply:
It depends on how much messages could be logged and how long may messages be (when we make smaller buffers or reduce number of buffers we may lose some data). For debugging it may be safer to keep these numbers bigger.